docs/clustering/logical-replication-setup.rst

```diff
@@ -0,0 +1,147 @@
+.. _logical_replication_setup:
+
+==================================================
+Logical replication setup between CrateDB clusters
+==================================================
+
+Logical replication is a method of data replication between multiple clusters.
+As a publish/subscribe model, it allows a publishing cluster to make certain
+tables available for subscription. Subscribing clusters pull changes from a
+publication and replay them on their side.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _requirements:
+
+Requirements
+============
+.. NOTE::
+
+  Logical replication is available in CrateDB 4.8 and later.
+
+To replicate data from one cluster to another, we need two CrateDB clusters.
+For this example, we spin them up locally as two single-node Docker
+containers.
+
+There exists no fixed designation of a cluster being a subscriber or publisher.
+One cluster can even take both roles for different tables. For the sake of
+simplicity, we will however refer here to the two clusters as
+``cluster-publishing`` and ``cluster-subscribing``.
+
+.. code-block:: console
+
+  sh$ docker run \
+        --name cluster-publishing \
+        --detach \
+        --publish 4201:4200 \
+        --publish 5433:5432 \
+        crate/crate:nightly-4.8.0-20220401 \
+        -Cdiscovery.type=single-node
+  sh$ docker run \
+        --name cluster-subscribing \
+        --detach \
+        --publish 4202:4200 \
+        --publish 5434:5432 \
+        crate/crate:nightly-4.8.0-20220401 \
+        -Cdiscovery.type=single-node
+
+We apply an offset to the ports of each cluster, so they don't overlap. The
+Admin UIs of ``cluster-publishing`` and ``cluster-subscribing`` are accessible at
+http://localhost:4201/ and http://localhost:4202/ respectively. All SQL
+statements discussed below can be executed via the corresponding Admin UI.
+
+As the subscriber pulls changes from the publisher, the publisher needs to accept
+incoming network connections from the subscriber. In our example setup, this is
+automatically given.
```

Comment:
```suggestion
incoming network connections from the subscriber. In our example setup, this is
automatically given because both containers are running on the same host.
For reaching each other, they will use the special ``host.docker.internal`` address.
```